core/ledger/ledger.go

```diff
@@ -177,6 +177,9 @@ func (ledger *Ledger) CommitTxBatch(id interface{}, transactions []*protos.Trans
 	ledger.blockchain.blockPersistenceStatus(true)
 
 	sendProducerBlockEvent(block)
+	if len(transactionResults) != 0 {
+		ledgerLogger.Debug("There were some erroneous transactions. We need to send a 'TX rejected' message here.")
```

Comment:
Is there an existing open issue? This is an important feature if the TransactionResult is to be removed. Assume it would use the events mechanism?